tests/components/local_file/test_config_flow.py

```diff
@@ -0,0 +1,70 @@
+"""Test the Local File config flow."""
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.local_file.const import DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from . import MOCK_CONFIG
+
+
+async def test_form(hass: HomeAssistant) -> None:
```

Comment:
I think you should create a `mock_setup_entry` fixture in a local `conftest.py` and auto-use in the config-flow tests.
```suggestion
pytestmark = pytest.mark.usefixtures("mock_setup_entry")


async def test_form(hass: HomeAssistant, mock_setup_entry: AsyncMock) -> None:
```